ApplicationLibCode/Commands/HoloLensCommands/VdeVizDataExtractor.cpp

```diff
@@ -417,7 +417,7 @@ QString VdeVizDataExtractor::createModelMetaJsonString( const std::vector<std::u
 
     ResInsightInternalJson::Json jsonCodec;
```

Comment:
This is now unused?

Reply:
This is currently used. Kristian did a job to replace use of QScript with QtCore/QJsonObject in 806c9af82beb755c50d7c393e967a570f5046536

The code is still used.

Reply:
The variable jsonCodec is now unused?

Reply:
Yes, I will fix the unused variable